docs/api/sql/Function.md

```diff
@@ -302,6 +302,24 @@ SELECT ST_AsEWKB(polygondf.countyshape)
 FROM polygondf
 ```
 
+## ST_AsEWKT
+
+Introduction: Return the Extended Well-Known Text representation of a geometry.
+EWKT is an extended version of WKT which includes the SRID of the geometry.
+The format originated in PostGIS but is supported by many GIS tools.
+If the geometry is lacking SRID a WKB format is produced.
+[Se ST_SetSRID](#ST_SetSRID)
```

Comment:
See ST_SetSRID not 'Se'.  If the geometry is lacking SRID a "WKT" format is produced.